app/models/gem_download.rb

```diff
@@ -33,6 +33,12 @@ def self.total_count
     end
   end
 
+  # version_id: 0 stores count for total gem downloads
+  # we need to find the second maximum
+  def self.most_downloaded_gem_count
+    count_by_sql "SELECT MAX(count) FROM gem_downloads WHERE count NOT IN (SELECT MAX(count) FROM gem_downloads)"
```

Comment:
Gem downloads are stored in columns `rubygem_id = gem_id, version_id = 0`. So, `version_id <> 0` won't work, however `rubygem_id <> 0` would.